homeassistant/components/scrape/config_flow.py

```diff
@@ -143,6 +142,49 @@ def validate_sensor_setup(
     return {}
 
 
+def validate_select_sensor(
+    handler: SchemaCommonFlowHandler, user_input: dict[str, Any]
+) -> dict[str, Any]:
+    """Validate sensor input."""
+    setattr(handler, "_idx", int(user_input[CONF_INDEX]))
```

Comment:
Maybe something like:
```suggestion
    handler.local_context["idx"] = int(user_input[CONF_INDEX])
```